app/templates/components/public/add-to-calender.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="ui list">
+  <div class="content">
+    <li>{{header-date @event.startsAt}} To {{header-date @event.endsAt}}</li>
```

Comment:
use general date helper and format it. And also I don't think it is needed here.

Reply:
i use it to give a similar format like given in the photo in the issue

Reply:
your format is not similar to this -
https://user-images.githubusercontent.com/1583873/93197307-c05ab780-f74b-11ea-8527-4cb703d739d9.png

Reply:
previously it was same u told me to use general date helper instead of header date
isn't?

Reply:
> your format is not similar to this -
> https://user-images.githubusercontent.com/1583873/93197307-c05ab780-f74b-11ea-8527-4cb703d739d9.png
previously it was same u told me to use general date helper instead of header date
isn't?

Reply:
it's wasn't same previously too, look closely

Reply:
can i make a new helper for the same format?